apps/docs/stories/components/SavedPaymentMethods/example.ts

```diff
@@ -0,0 +1,75 @@
+import { codeExampleHead } from '../../utils';
+
+export const codeExampleFull = `
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+
+${codeExampleHead(
+  'justifi-saved-payment-methods',
+  `<style>
+      /** These are the available parts for styling the component. */
+      ::part(text) {
+        // text styles
+      }
+
+      ::part(text-danger) {
+        // text danger styles
+      }
+      
+      ::part(label) {
+        // label styles
+      }
+
+      ::part(input-radio) {
+        // input radio styles
+      }
+
+      ::part(input-radio-focused) {
+        // input radio focused styles
+      }
+
+      ::part(input-radio-checked) {
+        // input radio checked styles
+      }
+
+      ::part(input-radio-checked-focused) {
+        // input radio checked focused styles
+      }
+
+      ::part(input-radio-invalid) {
+        // input radio invalid styles
+      }
+    </style>`
+)}
+
+<body>
+  <justifi-modular-checkout
+    auth-token="authToken"
+    account-id="acc_123"
+    checkout-id="ch_123"  
+    save-payment-method="true"
```

Comment:
I don't think a checkbox would work in this use case. Could you please remove that?